_data-prepper/monitoring.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: default
+title: Monitoring
+nav_order: 12
+---
+
+# Monitoring Data Prepper with Metrics
+
+Monitoring metrics in Data Prepper is instrumented using [Micrometer](https://micrometer.io/). There are two types of metrics: JVM and system metrics, and plugin metrics. [Prometheus](https://prometheus.io/) is used as the default metrics backend.
+
+## JVM and system metrics
+
+JVM and system metrics are runtime metrics used to monitor Data Prepper instances. They include metrics for classloaders, memory, garbage collection, threads, and others. For details, [JVM and system metrics](https://micrometer.io/docs/ref/jvm) for details. 
+
+### Naming
+
+JVM and system metrics in Data Prepper follow pre-defined names in Micrometer, such as `jvm_classes_loaded`, and `jvm_memory_used`. 
```

Comment:
Does this naming convention relate to the "Naming Meters" guidance in Micrometer documentation?
https://micrometer.io/docs/concepts#_naming_meters
Or is this a different convention?